src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+
+type Props = {
+  changeQuery: (query: string) => void,
+  isCompleted: boolean,
+  numberActive: number,
+};
+
+enum StatusTodos {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
+}
+
+export const Footer: React.FC<Props> = ({
+  changeQuery,
+  isCompleted,
+  numberActive,
+}) => {
+  const [status, setStatus] = useState('All');
+
+  const handleClickAll = () => {
+    changeQuery(StatusTodos.all);
+    setStatus(StatusTodos.all);
```

Comment:
Seems that you don't need to create a state for the status here. It repeats the query state from the App component 
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/99403198/98e23a93-3c94-4f4c-a2f4-604632974d8b)